app/src/main/java/com/example/taskyy/ui/LoginScreen.kt

```diff
@@ -61,7 +61,7 @@ fun LoginScreen(state: LoginState, onEvent: (LoginEvent) -> Unit, navController:
                     .fillMaxHeight(.75f)
                     .padding(30.dp)
             ) {
-                CreateEmailField(state.isEmailValid, onEvent, state.email)
+                CreateEmailField(state.isEmailValid, (String) -> Unit,  state.email)
```

Comment:
Here, the syntax is not correct. `(String) -> Unit` is correct when defining a lambda inside the parameter list, but when creating a lambda reference, you can just to this with curly braces. In this example, it would need to be like this:
```
CreateEmailField(
    state.isEmailValid,
    { email -> onEvent(LoginEvent.OnEmailChange(email) },
    state.email
)
```